### PR TITLE
Update code due to the change of Office Add-in page

### PR DESCRIPTION
**Pull request by ChunLi0819**

### CODE REVIEW

File: DevOfficeComTest/TestFramework/OfficeAddInPage/CardBuild.cs [hunk 1]
```diff
@@ -19,14 +19,20 @@ public void StartBuilding()
                 case Product.Outlook:
                     {
                         var buildBtn = Browser.Driver.FindElement(By.Id("more-playground"));
```
Comment (JingyuShao) [inline DevOfficeComTest/TestFramework/OfficeAddInPage/CardBuild.cs]: there is no more-playground button on the UI anymore.

File: DevOfficeComTest/TestFramework/OfficeAddInPage/CardBuild.cs [hunk 2]
```diff
@@ -19,14 +19,20 @@ public void StartBuilding()
...
+                        {
+                            Browser.Click(buildBtn);
+                        }
                         break;
                     }
                 case Product.PowerPoint:
                 case Product.Word:
                     {
                         var buildBtn = Browser.Driver.FindElement(By.Id("build-downloadFromGithub"));
-                        Browser.Click(buildBtn);
```
Comment (JingyuShao) [inline DevOfficeComTest/TestFramework/OfficeAddInPage/CardBuild.cs]: There is no build-downloadFromGithub button on the UI anymore.

Comment (ChunLi0819) [inline DevOfficeComTest/TestFramework/OfficeAddInPage/CardBuild.cs]: This button and above exist in the divs buildDefault and buildWithGithubSample. Even though they are set as display:none, the buttons on them still can be clicked by cases. Will delete this part directly

File: DevOfficeComTest/TestFramework/OfficeAddInPage/CardBuild.cs (hunk 1 above)
Comment (ChunLi0819) [inline DevOfficeComTest/TestFramework/OfficeAddInPage/CardBuild.cs]: Will delete this part directly

**Comment by ChunLi0819** [comment]
Close this request. Will update based on pull request #25

**Pull request closed without merging**